Common/src/main/resources/META-INF/common-plugin.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" ?>
 <idea-plugin require-restart="true" url="https://tabnine.com">
-    <name>Tabnine AI Code Completion- JS Java Python TS Rust Go PHP &amp; More</name>
+    <name>Tabnine: AI Code Completion &amp; Chat in Java JS/TS Python &amp; More</name>
```

Comment:
this name doesn't really matter, its in the common XML so its not determining the actual name.
i just copy-pasted from the GA plugin here for the sake of aesthetics